### ISSUE TITLE
Use searxng or duckduckgo instead bing

### ISSUE THREAD

**Issue by alsoasnerd** [shadowfax92/Fyin#1]
I can do it?

**Comment by shadowfax92** [comment]
Yes please feel free to! 🙂

**Comment by alsoasnerd** [comment]
It's just change the BING_ENDPOINT for my searxng URL? I'm not familiar with compatibility between bing and searxng.

Sorry for my English.

**Comment by shadowfax92** [comment]
Ah my bad, I thought you were planning to add support.

No sorry, today it's not supported but it's on my radar to add support.

Will take a look at the docs and add support in coming weeks - https://github.com/searxng/searxng

**Comment by quantumalchemy** [comment]
yes please setup for searxng or duckduckgo  . as bing now requires a cc for the free tier 
if I knew rust I would pull it but.. looking at the code doesnt look to bade proly quick map the params to  searxng or ddg
Great work!
